js/window.js

```diff
@@ -206,4 +206,30 @@ form.addEventListener('submit', (event) => {
   if (!emailValid) {
     event.preventDefault();
   }
-});
\ No newline at end of file
+});
+/* save form data */
+const userName = form.elements.name;
+const userMail = form.elements.email;
+const userMessage = form.elements.message;
+function populateStorage() {
+  localStorage.setItem('name', form.elements.name.value);
+  localStorage.setItem('mail', form.elements.email.value);
+  localStorage.setItem('message', form.elements.message.value);
+}
+function setForm() {
+  const currentUserName = localStorage.getItem('name');
+  const currentUserEmail = localStorage.getItem('mail');
```

Comment:
- [ ] **REQUIRED:** Please, in order to be used after retrieve from the local storage the data object must be turned back into an object using `JSON.parse`